collections/ansible_collections/purestorage/flasharray/README.md

```diff
@@ -47,6 +50,7 @@ All modules are idempotent with the exception of modules that change or set pass
 - purefa_phonehome - manage the phonehome setting for the FlashArray
 - purefa_pod - manage ActiveCluster pods in FlashArrays
 - purefa_pod_replica - manage ActiveDR pod replica links in FlashArrays
+- purefa_policy - manage FlashArray managed policies for NFS, SMB and snapshot
```

Comment:
snapshot**b**?

Reply:
Yes - it will cover snapshot policy for filesystems as well

Reply:
I ment snapshots instead of snapshot?